src/users/get-users.ts

```diff
@@ -25,13 +25,26 @@ export function getUsers(authorizationExtensionUrl: string, accessToken: string)
       ...getUsersDefaultOptions,
       ...options
     };
-    return get({
-      accessToken,
-      url: `${authorizationExtensionUrl}/users`,
-      queryParams: {
-        page: `${options.page}`,
-        per_page: `${options.perPage}`,
-      }
-    });
+
+    const getPaged: any = function(page: number) {
+      return get({
+        accessToken,
+        url: `${authorizationExtensionUrl}/users`,
+        queryParams: {
+          page: `${options.page}`,
+          per_page: `${options.perPage}`,
+        }
+      }).then((result: any) => {
+        const total_pages = Math.ceil(result.total / options.perPage);
```

Comment:
can you pull this out into a function that gets called both here and in get-group-members?